misk/web/@misk/common/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@misk/common",
+  "version": "0.0.1",
+  "description": "Microservice Kontainer Common Libraries & Styles",
+  "author": "Square/Misk Authors (https://github.com/square/misk/graphs/contributors)",
```

Comment:
are all of these required? Can ya remove the optional ones?

Reply:
👍